pkgs/os-specific/linux/ledger-udev-rules/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, fetchFromGitHub, bash }:
+stdenv.mkDerivation {
+  name = "ledger-udev-rules";
+
+  src = fetchFromGitHub {
+    owner = "LedgerHQ";
+    repo = "udev-rules";
+    rev = "681cd1346f2dd5bc3ab19a7985caabbc97e1861b";
+    sha256 = "0g9xb98mpv4rimqbcaqs3qcdqalf6v1h1wjcafgbx9vslz9x4a14";
+  };
+
+  installPhase = ''
+    mkdir --parents "$out/lib/udev/rules.d"
+    cat add_udev_rules.sh | grep echo | sed "s_/etc_$out/lib_" | sed "s_hw1_ledger_" | sed 's_MODE=\\"0660\\",__' | sed 's_GROUP=\\"plugdev\\"_TAG+=\\"uaccess\\"_' > install.sh
```

Comment:
Could you maybe ask upstream to provide the rules in a separate file?

Reply:
I'll try.

Reply:
Submitted https://github.com/LedgerHQ/udev-rules/pull/1 upstream.

Reply:
Could we just embed these rules?

Reply:
I did it this way because I was trying to follow the example of an existing package (libu2f-host), but I don't have an opinion. I'd be happy to amend the PR to embed the rules if that's preferable.

Reply:
I think the script is preferable – since upstream does not provide the file, we would need to convert the file with each update anyway.

Reply:
It got merged!